packages/eslint-plugin-shopify/lib/config/rules/lodash.js

```diff
@@ -0,0 +1,84 @@
+// see https://github.com/wix/eslint-plugin-lodash
+
+module.exports = {
+  // Possible errors
+
+  // Use or avoid thisArg for Lodash method callbacks, depending on major version.
+  'callback-binding': 'warn',
+  // Use value returned from collection methods properly.
+  'collection-method-value': 'warn',
+  // Always return a value in iteratees of Lodash collection methods that aren't forEach.
+  'collection-return': 'error',
+  // Do not use .value() on chains that have already ended (e.g. with max() or reduce()) (fixable)
+  'no-double-unwrap': 'error',
+  // Do not use superfluous arguments on Lodash methods with a specified arity.
+  'no-extra-args': 'error',
+  // Prevent chaining without evaluation via value() or non-chainable methods like max().
+  'unwrap': 'error',
+
+  // Stylistic issues
+
+  // Enforce a specific chain style: explicit, implicit, or explicit only when necessary.
+  'chain-style': ['warn', 'explicit'],
+  // Prefer identity shorthand syntax
+  'identity-shorthand': ['warn', 'never'],
+  // Prefer matches property shorthand syntax
+  'matches-prop-shorthand': ['warn', 'never'],
+  // Prefer matches shorthand syntax
+  'matches-shorthand': ['warn', 'never'],
+  // Do not use .commit() on chains that should end with .value()
+  'no-commit': 'warn',
+  // Prevent chaining syntax for single method, e.g. _(x).map().value()
+  'no-single-chain': 'warn',
+  // Enforce a specific path style for methods like get and property: array, string, or arrays only for deep paths.
+  'path-style': ['warn', 'string'],
+  // Prefer a Lodash chain over nested Lodash calls
+  'prefer-chain': 'warn',
+  // Prefer _.compact over _.filter for only truthy values.
+  'prefer-compact': 'warn',
+  // Prefer _.filter over _.forEach with an if statement inside.
+  'prefer-filter': 'warn',
```

Comment:
I think our general approach on error versus warn is, if this is probably a coding error or an absolutely horrendous style thing (thing `with`/ labels/ `eval`), that's an error. If it's just a bad practice or something we want to dissuade, that's a warn. IMO, this probably falls into the second category.